scripts/bump.sh

```diff
@@ -2,11 +2,23 @@
 
 set -eu
 
+# For local development, in production, the environment will be set though GH actions and GH secrets
+if [ -f ".envrc" ]; then
+    echo "Loading .envrc"
+    # shellcheck disable=SC1091
+    . .envrc
+else
+    echo "No .envrc"
+fi
+
+
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-NPM_VERSION=$2
-echo "NPM_VERSION: $NPM_VERSION"
+PRISMA_VERSION=$(sh scripts/prisma-version.sh "$RELEASE_CHANNEL")
+echo "UPDATING to $PRISMA_VERSION"
+echo "PRISMA_VERSION: $PRISMA_VERSION"
```

Comment:
I don't fully understand this. `UPDATING TO` and `PRISMA_VERSION` shouldn't be the same variable right?

Reply:
think it should. This line was in `check-update` script before and was called directly before running `bump.sh`. In `check-update.sh`, this variable was called `NPM_VERSION` and given as an argument to `bump.sh`. I removed the argument and instead got the variable directly from the script.

Reply:
But still, one of these echo statements is enough right. Since they convey the same information? Do you agree?

Reply:
yes, I'll remove one of them